app/views/layouts/application.html.erb

```diff
@@ -16,30 +16,30 @@
   <body class="bg-gray-blue black-70 f6 f5-ns">
     <nav class="bg-white shadow pt3 ph2 mb4">
 
-      <div class="mw7 center flex-ns justify-between">
-        <div class="pv2 b">
-          💙 Basic Rights
+      <div class="mw7 center flex-l justify-between">
+        <div class="b tracked-tight pl2 pl0-l pv2">
+          <%= link_to '💙Basic Rights', root_path %>
         </div>
-        <div class="dib mb4">
-          <%= link_to 'Home', root_path, class: "link hover-blue dib pa2" %>
-          <%= link_to 'Organizations', organizations_path, class: "link hover-blue dib pa2" %>
-          <%= link_to 'About', about_path, class: "link hover-blue dib pa2" %>
-          <%= link_to 'Sign in', user_session_path unless current_user %>
-          <%= link_to 'Sign Out', destroy_user_session_path, class: "link hover-blue dib pa2", method: :delete if current_user %>
+        <div class="dib mb3">
+          <%= link_to 'Requests', needs_path, class: "link hover-blue dib pa2 f6 f5-l" %>
+          <%= link_to 'Offers', surplus_path, class: "link hover-blue dib pa2 f6 f5-l" %>
+          <%= link_to 'Orgs', organizations_path, class: "link hover-blue dib pa2 f6 f5-l" %>
+          <%= link_to 'About', about_path, class: "link hover-blue dib pa2 f6 f5-l" %>
+          <%= link_to 'Admin', admin_path, class: "link hover-blue dib pa2 f6 f5-l" %>
```

Comment:
This is a comment to me so that I create a helper method so that the Admin link only appears if the current user has access to that page. After the method is created I will update by appending the conditional `if admin?`

Reply:
Created issue #85 for this.